pages/common/ninja.md

```diff
@@ -0,0 +1,20 @@
+# ninja
+
+> Build system designed to be fast.
+> More information: <https://ninja-build.org/manual.html>.
+
+- Build in current directory:
+
+`ninja`
+
+- Build a program in DIR:
+
+`ninja -C DIR`
+
+- Show help:
+
+`ninja -h`
+
+- Show targets like install or unistall:
```

Comment:
I think just saying "Show targets" is enough ?

Reply:
I am not so sure about that, as it was not immediately clear to me what a target is. I think the example helps.

Reply:
Right. Perehaps this would read better then?
```suggestion
- Show targets (e.g. `install` and `uninstall`):
```